packages/flutter/lib/src/widgets/statistics_overlay.dart

```diff
@@ -36,6 +36,7 @@ enum StatisticsOption {
   visualizeEngineStatistics,
 }
 
+/// Displays performance statistics.
```

Comment:
Should explain what they look like and how to read them, or point to a wiki page or flutter.io page doing the same.